modules/understand-admission-controller-enforcement.adoc

```diff
@@ -23,5 +23,5 @@ Also, consider API timeouts in any custom automation you might be using.
 * The admission controller might fail if there are connectivity issues between the Kubernetes or {ocp} API server and {product-title-short} Sensor.
 To resolve this issue, delete the `ValidatingWebhookConfiguration` object as described in the disabling admission controller enforcement section.
 //link to Disable admission controller enforcement
-* If you have deploy-time enforcement enabled for a policy and you enable the admission controller, {product-title-short} attempts to block deployments that violate the policy.
+* If you have deploy-time enforcement enabled for a policy and you enable the admission controller, {product-title-short} attempts to block deployments that violate the policy. (Please note that `Reasses all` will not scale down existing deployments.)
```

Comment:
Hi, a couple of things:
1) Avoid "please" in technical documentation per IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=medium-tone "In technical information, avoid terms of politeness, such as please and thank you. These terms are inappropriate or offensive in some cultural contexts."
2) For a note, we use the admonition format: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#admonitions

so it would be:

```
+
[NOTE]
====
`Reasses all` will not scale down existing deployments.
====

```

Reply:
Done